slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "MMRM and ASA SWE - R Adoption Series"
+author: "Ben Arancibia and Yoni Sidi"
+format: 
+  revealjs:
+    theme: dark
+    incremental: false
+    logo: https://github.com/RConsortium/asa-biop-swe-wg/raw/main/sticker/sticker-1200.png
+    slide-number: c/t
+title-slide-attributes: 
+  data-background-image: https://github.com/RConsortium/asa-biop-swe-wg/raw/main/sticker/sticker-1200.png
+  data-background-size: 20%
+  data-background-opacity: "0.5"
+  data-background-position: 80% 10%
+editor: visual
+---
+
+## Agenda
+
+-   Introduction and Overview of AWA SWE
```

Comment:
let's consistently use:
```suggestion
-   Introduction and Overview of SWE WG
```
I would not emphasize too much on the ASA BIOP association (at least for the abbreviation) since we might soon also be associated with EFSPI